docs/topics/multiplatform/multiplatform-hierarchy.md

```diff
@@ -153,11 +156,13 @@ The resulting hierarchical structure will look like this:
 
 ```kotlin
 kotlin {
+    ios()
```

Comment:
As far as I can see, this section wants to show a manual configuration equivalent to the usage of `ios`, so you can't use `ios` here (it's kinda self-referential)

Use `iosArm64()` and `iosX64()` instead

Reply:
I guess we have a confusion here. The section is about target shortcuts actually. We introduce an example with the ios() shortcut first, then show a diagram, and then explain the resulting hierarchy. So I'm not sure what's the best way to write the code snippet for the resulting hierarchy in this case